src/routes/(admin)/account/+page.server.ts

```
@@ -0,0 +1,81 @@
+		const token = getSessionTokenCookie();
+
+		const form = await superValidate(request, zod(editUserSchema));
+		if (!form.valid) {
+			return fail(400, {
+				form
+			});
+		}
+		console.log(JSON.stringify(form.data));
+
+		let res: Response;
+		const updateData: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)